packages/angular/projects/ui-angular/src/lib/components/authenticator/components/federated-sign-in/federated-sign-in.component.ts

```diff
@@ -27,7 +27,7 @@ export class FederatedSignInComponent implements OnInit {
   constructor(private authenticator: AuthenticatorService) {}
 
   ngOnInit(): void {
-    const { socialProviders } = this.authenticator.context?.config;
+    const socialProviders = this.authenticator.context?.config?.socialProviders;
```

Comment:
Eslint was complaining at this because `config` could be undefined and destructuring from it is not desirable.

Reply:
This is the only change I made in this merge.